src/session.c

```diff
@@ -0,0 +1,17 @@
+#include "session.h"
+#include "packet.h"
+
+bool ddnet_seq_in_backroom(uint16_t sequence, uint16_t ack) {
+	int32_t bottom = ack - (DDNET_MAX_SEQUENCE / 2);
+	if(bottom < 0) {
+		if(sequence <= ack) {
+			return true;
+		}
+		if(sequence >= (bottom + DDNET_MAX_SEQUENCE)) {
```

Comment:
```suggestion
		if((int32_t)sequence >= (bottom + DDNET_MAX_SEQUENCE)) {
```
Could you add `(int32_t)` where signed and unsigned values are compared, sometimes it can be tricky, for example: 
```C
int main() {
  uint32_t sequence = 15;
  int32_t bottom = -1;

  if (sequence > bottom) {
    printf("owo");
  }

  return 0;
}
```
doesn't print `owo`, but
```C
int main() {
  uint16_t sequence = 15;
  int32_t bottom = -1;

  if (sequence > bottom) {
    printf("owo");
  }

  return 0;
}
```
does print 😬

Reply:
lerato level C knowledge right there